components/Panel/Panel.css

```diff
@@ -1,6 +1,5 @@
 .root {
+  composes: bs br from '../../globals/modifiers.css';
```

Comment:
Ohhh now I know what you mean. The Panel radius & shadow wouldn't show on the style guide.

The culprit was we're using `composes` above (formally `extends` in css-loader 0.14.x) — but it made no difference when using the styles in PactApp or newAccount as they both had css-loader 0.15.x.

Fixed now anyways.